quirks/websites-with-shared-credential-backends.json

```diff
@@ -198,5 +198,9 @@
     [
         "lookmark.io",
         "lookmark.link"
+    ],
+    [
+        "origin.com",
+        "myaccount.ea.com"
```

Comment:
When trying to access `myaccount.ea.com` it doesn't seem to resolve. Also signing in to `origin.com` prompts a new window that references `signin.ea.com`. (`ea.com` also redirects you to sign in at `signin.ea.com`)